beesdoo_product/models/beesdoo_product.py

```diff
@@ -64,19 +64,21 @@ class BeesdooProduct(models.Model):
         translate=True,
     )
 
-    top_supplierinfo_id = fields.Many2one(
+    main_supplierinfo_id = fields.Many2one(
         comodel_name="product.supplierinfo",
-        compute="_compute_main_seller_id",
+        compute="_compute_main_supplierinfo_id",
         store=True,
     )
```

Comment:
I name it wrong first time. As it is not the first supplierinfo but realy the main one as it's computed via the `_get_main_supplier_info()` method.